### PR TITLE
chore: adds order initial documentation

### PR DESCRIPTION
**Pull request by jpinho**
Updates the docs, with summary documentation for Orders. Including an SDK example.

docs

### CODE REVIEW

File: docs/pricing/orders.mdx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+<img style={{ boxShadow: '#a2a2a2 0px 0px 25px -8px', borderRadius: '5px' }} src="/img/order-detailsview.png" alt="Order Details View" />
+
+## Managing Orders via Pricing API
+
+Using our [Entity API](/docs/entities/entity-api) we can directly manage Orders, although very rapidly we realise that an Order has more to it than just a collection of Products and Prices. We needed to be able to manage the Order's lifecycle, and we needed to be able to manage the Order's state. We also needed to be able to recompute the order's product and price totals, along with the order price and product relations. Using just our Entity API would not suffice here since there is additional logic required to fully manage an order.
+
+To fullfil this purpose we have designed the [Pricing API](/docs/pricing/pricing-api), which is a wrapper around the Entity API. The Pricing API allows us to manage Orders, and it also allows us to manage the Order's lifecycle and state. The Pricing API also allows us to recompute the order's product and price totals, along with the order price and product relations.
+
+
+### Updating an Order using `epilot-sdk`
```
Comment (anttiviljami) [inline docs/pricing/orders.mdx:20]: Creating Orders via the API is the most common use case, so I’d consider using that as the example here.

**Review by anttiviljami — APPROVED** [review]
LGTM!

File: docs/pricing/orders.mdx (hunk 1 above)
Comment (jpinho) [reply docs/pricing/orders.mdx]: I've added a way to provide both examples with an API example and an SDK example.

Screen Shot 2022-09-20 at 17 04 25

**Comment by jpinho** [comment]
To merge this I'm just missing 1 thing, I've bumped some of our dependencies to v2.1.0 (away from beta versions) but I'm struggling with the `apiDocComponents` (https://redocusaurus.vercel.app/docs/guides/migrating-to-v1) that has been deprecated and I can't seem to be able to make it work... It's the last missing bit 😭

**Pull request merged**